README.md

```diff
@@ -1,2 +1,110 @@
-# Air-Hockey-Robot
-Srishti 2020
+# AIR HOCKEY ROBOT
+### Shrishti'20, IITR
+## Image of the project
```

Comment:
Remove this sub-heading and remove '1.' at the beginning from "1. Table isometric view"